src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -54,10 +58,25 @@ public QuoraProfileScraper(String _query) {
         this.query = _query;
     }
 
-    public QuoraProfileScraper(String _query, String _extra) {
+    public QuoraProfileScraper(String _query, Map<String, String> _extra) {
         this();
+        this.setExtra(_extra);
         this.query = _query;
-        this.extra = _extra;
+    }
+
+    public QuoraProfileScraper(Map<String, String> _extra) {
+        this();
+        this.setExtra(_extra);
+    }
+
+    protected void setParam() {
+        if("".equals(this.getExtraValue("type"))) {
+            this.typeList = Arrays.asList(this.getExtraValue("type").trim().split("\\s*,\\s*"));
+        } else {
+            this.typeList = new ArrayList<String>();
+            this.typeList.add("all");
+            this.setExtraValue("type", String.join(",", this.typeList));
```

Comment:
typeList is ArrayList that stores strings. if there is no get parameter, then `all` shall be set by default. 
So these lines does the task 
```
this.typeList = new ArrayList<String>();
 this.typeList.add("all");
this.setExtraValue("type", String.join(",", this.typeList));
```